ads/google/a4a/test/test-utils.js

```diff
@@ -479,8 +479,12 @@ describe('Google A4A utils', () => {
       });
       const ampAdElem = env.win.document.createElement('amp-ad');
       prevContainer.appendChild(ampAdElem);
+      const ValidAdContainerTypeValues =
+          Object.keys(ValidAdContainerTypes).map(function(key) {
```

Comment:
camelCase

Reply:
nvm, guess it was already PascalCase prior to changes

Reply:
Yeah, didn't want to edit the rest of the file for this fix.